flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/test/program/SinkTestStep.java

```diff
@@ -63,6 +69,8 @@ public final class SinkTestStep extends TableTestStep {
         this.expectedAfterRestore = expectedAfterRestore;
```

Comment:
Add an additional `if` check to verify that `expectedBatchRows` and not mixed with strings and Before/After fields.

Reply:
True, you are right. Then we should check the mixture of all Rows vs all Strings.

Reply:
Ok, pushed a commit to address this.